.github/workflows/python-poetry-tests.yml

```diff
@@ -23,6 +28,8 @@ jobs:
   test:
     name: 🔬 Test (🐍 ${{ inputs.python-version }})
     runs-on: ubuntu-latest
+    container:
+      image: ${{ inputs.container-image }}
```

Comment:
What happens if no `container-image` variable is set?

Reply:
In principle, if the variable is empty it just skips it. I was about to confirm this next but it looks like the docker pulls fail https://github.com/NNPDF/nnusf/actions/runs/3138407519/jobs/5097703520

Reply:
We should perhaps remove the authentication when pulling the image?

Reply:
There is no authentication: it is a public package, and the workflow should be able to access it without further ado.

Here it is actually working: https://github.com/N3PDF/pineappl/blob/master/.github/workflows/wheels.yml

Reply:
This is what I thought as well but it looks like one needs to authenticate to the container registry. Otherwise, I don't know what's going on. 

If you try to run the action and it doesn't work then it must be something else.